UPGRADING.rst

```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
 +---------------------------------+----------------------------------+--------------------+
 | ``web_tls_key_password``        | ``http_tls_key_password``        | Empty              |
 +---------------------------------+----------------------------------+--------------------+
+
+Plugins moved into the Graylog server
```

Comment:
s/moved/merged/

(Move indicates that they're still plugins, but in a different place)